### DISCUSSION TITLE
error win i use it to open ppt file, the error code from win32 is -2147352570

### DISCUSSION THREAD

**Discussion by x19990416** [mhammond/pywin32#2551] [General]
Note that issues in this repository are only for bugs or feature requests in the pywin32.

**If you need support or help using this package, please follow [these instructions](https://github.com/mhammond/pywin32/blob/master/README.md#support)** - support or help requests will be closed without comment.

For all bugs, please provide the following information.

* Expected behavior and actual behavior.

* Steps to reproduce the problem.

* Version of Python and pywin32

pywin32  version:  224

when i use it to open ppt file, there is some exception。see the detail from the image
1st img，the py code how i use pywin32 。
2rd img, the error from the pywin32 (`win32com\client\dynamic.py`)
3rd img, the error where i debug info.

![image](https://user-images.githubusercontent.com/9440757/56491035-c4b08000-6519-11e9-99c4-218b266f8906.png)

![image](https://user-images.githubusercontent.com/9440757/56491014-bb271800-6519-11e9-983d-daf896b745df.png)

![image](https://user-images.githubusercontent.com/9440757/56491012-b3677380-6519-11e9-9657-06f3ab046ff9.png)

**Comment by mhammond** [comment]
Please re-read the first 2 lines in your report - this is a support request, not a bug report.
